system/xmllib/xpathprocessor.hpp

```diff
@@ -152,11 +152,11 @@ extern "C" XMLLIB_API IXpathContext*  getXpathContext(const char * xmldoc, bool
 #define ESDLScriptCtxSection_ScriptRequest "script_request"
 #define ESDLScriptCtxSection_ScriptResponse "script_response"
 
-interface IEsdlScriptContext : extends IInterface
+interface ISectionalXmlDocModel : extends IInterface
 {
     virtual IXpathContext* createXpathContext(IXpathContext *parent, const char *section, bool strictParameterDeclaration) = 0;
     virtual IXpathContext *getCopiedSectionXpathContext(IXpathContext *parent, const char *tgtSection, const char *srcSection, bool strictParameterDeclaration) = 0;
-    virtual void *queryEspContext() = 0;
+    virtual void *queryUserData() = 0;
```

Comment:
Is this ever even called anymore.  I see queryEspContext() used.  And I see the userdata that is stored in the xpath context being used.  But I don't see where this userData is used anymore.

Reply:
It isn't called in relation to ESDL script because of how the script context wraps the doc model. I did recognize that it is unused when I looked for instances to rename, and didn't find it being used. It can be removed if we don't care about the model being used in any other way.

Reply:
Yeah I'd rather remove it until it is needed.